src/status_im/chat/views/message/message.cljs

```diff
@@ -236,22 +227,31 @@
   (letsubs [{:keys [photo-path]} [:get-current-account]]
     (photo from photo-path)))
 
+(defview message-author-name [from]
+  (letsubs [username    [:contact-name-by-identity from]]
+    [react/text {:style {:font-size      12
```

Comment:
Can we move this to styles? Or maybe some component? ;)

Reply:
agree, it's better to move it to styles first